reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedIssueType.kt

```diff
@@ -22,6 +22,6 @@ package org.ossreviewtoolkit.reporter.reporters.evaluatedmodel
 /**
  * The possible types of an [EvaluatedOrtIssue].
  */
-enum class EvaluatedOrtIssueType {
+enum class EvaluatedIssueType {
```

Comment:
Commit message: "the the" -> "with the"
Technically this is also a breaking change for programmatic use as the class is not internal.